.github/actions/latest-stable-tag/__tests__/get-latest-tag.test.js

```diff
@@ -0,0 +1,26 @@
+import { getLatestTag } from '../get-latest-tag.js';
+
+describe('getLatestTag', () => {
+
+  const SIMPLE_TAGS = [ '1.0.0', '1.0.1', '1.1.0', '1.2.0', '1.2.1' ];
+  const TAGS_WITH_PATCH_ON_TWO_DIGITS = [ '1.0.0', '1.1.0', '1.2.0', '1.2.2', '1.2.10' ];
```

Comment:
> Is it because "10" has 2 digits?

Yes. To ensure we compare version numbers correctly, based on the numeric values of the components (and not just alphabetically), i.e. `1.2.2 < 1.2.10`

> TAGS_WITH_TWO_DIGIT_PATCH

Works for me!